src/http.rs

```diff
@@ -0,0 +1,22 @@
+use curl;
+use oauth2::*;
+use app::App;
+
+pub fn github(app: &App, url: &str, auth: &Token)
```

Comment:
I made two fns, one for querying, one for parsing, so that you can intermediate evaluation of the request as desired.